binary_transparency/firmware/internal/ftmap/aggregate.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Could you please add a one liner comment (similar to ftmap.go line 15) stating the intent of the file

Reply:
The way that this works in go is that the top level comment is for the package, not for the file. If I put something at that level here it will fight with the documentation in `pipeline.go` which better describes this package.

Is the choice of file confusing? The log stuff is in `log.go`, thus `aggregate.go` seems like a natural home for the aggregation related sub-pipeline.

Reply:
Ok, understood!